setup.cfg

```
@@ -202,6 +202,8 @@ filterwarnings =
     ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning:zeep.utils
     # Deprecation warning added in NumPy 1.24, triggered by scikit-image
     ignore:.*np.bool8.*is a deprecated alias.*:DeprecationWarning
+    # Warnings for sqlalchemy 2.0. For now we pin the version and ignore this warning
+    ignore:Deprecated API features detected:sqlalchemy.exc.MovedIn20Warning
```

This would probably be enough?
```suggestion
    ignore:Deprecated API features detected*SQLAlchemy 2.0
```